config/boards/radxa-zero3.conf

```diff
@@ -3,14 +3,19 @@ BOARD_NAME="Radxa ZERO 3"
 BOARDFAMILY="rk35xx"
 BOARD_MAINTAINER="Radxa"
 BOOTCONFIG="radxa-zero3-rk3566_defconfig"
-KERNEL_TARGET="legacy,vendor"
+KERNEL_TARGET="legacy,vendor,edge"
 FULL_DESKTOP="yes"
 BOOT_LOGO="desktop"
 BOOT_FDT_FILE="rockchip/rk3566-radxa-zero3.dtb"
 IMAGE_PARTITION_TABLE="gpt"
 BOOT_SCENARIO="spl-blobs"
 BOOTFS_TYPE="fat"
 
+
+function post_family_config_branch_edge__use_mainline_dtb_name() {
+	BOOT_FDT_FILE="rockchip/rk3566-radxa-zero-3e.dtb"
```

Comment:
take note about both the additional `-` and the exact board definition `3e` which breaks `3w` compatibility.

Reply:
Did you try to `unset BOOT_FDT_FILE` or `BOOT_FDT_FILE=""` here? Mainline U-Boot should autodetect what board is booting and set U-Boot `fdtfile` env var accordingly, may help improve support for `3w`.